src/firefly/js/metaConvert/LsstSdssRequestList.js

```diff
@@ -12,60 +12,78 @@ import {getCellValue} from '../tables/TableUtil.js';
+    const field= padStart(getCellValue(table, rowIdx, 'field'), 4, '0');
+    const camcol= getCellValue(table, rowIdx, 'camcol');
+    const filterId= getCellValue(table, rowIdx, 'filterId');
 
-const DAX_URL= 'http://lsst-qserv-dax01.ncsa.illinois.edu:5000';
-const bandMap= {u:0, g:1,r:2,i:3, z:4};
+    const sr= new ServerRequest('LSSTImageSearch');
+    sr.setParam('run', `${run}`);
+    sr.setParam('camcol', `${camcol}`);
+    sr.setParam('filterId', `${filterId}`);
+    sr.setParam('field', `${field}`);
 
+    return (plotId, title, filterName) => {
 
+        sr.setParam('filterName', `${filterName}`);
+        return makeWebRequest(sr,plotId,  title);
+    };
+}
 
+/**
+ * This method returns a WebRequest object
+ * @param sr - {Object}
+ * @param plotId - {String}
+ * @param title - {String}
+ * @returns {a WebRequest object}
+ */
 
-function makeUrlPlotRequest(url, title, plotId, rangeValues) {
-    const r= WebPlotRequest.makeURLPlotRequest(url);
+function makeWebRequest(sr,  plotId, title) {
+    const r  = WebPlotRequest.makeProcessorRequest(sr, 'lsst');
+    const rangeValues= RangeValues.makeRV({which:SIGMA, lowerValue:-2, upperValue:10, algorithm:STRETCH_LINEAR});
     r.setTitleOptions(TitleOptions.NONE);
     r.setTitle(title);
     r.setPlotId(plotId);
     r.setMultiImageIdx(0);
-    r.setPreferenceColorKey('lsst-sdss-color-pref');
+    r.setPreferenceColorKey('lsst-coadd-color-pref');
     r.setZoomType(ZoomType.TO_WIDTH);
     r.setInitialRangeValues(rangeValues);
     return r;
-
 }
+/**
+ * This method builds the Coadd search string
+ * @param table - {Object }
+ * @param rowIdx - {int}
+ * @returns {Function}
+ */
+function makeCoadReqBuilder(table, rowIdx) {
 
+    const bandMap= {u:'0', g:'1',r:'2',i:'3', z:'4'};
 
-
-function makeCcdReqBuilder(table, rowIdx) {
-    const rangeValues= RangeValues.makeRV({which:SIGMA, lowerValue:-2, upperValue:10, algorithm:STRETCH_LINEAR});
-    const run= getCellValue(table, rowIdx, 'run');
-    const field= padStart(getCellValue(table, rowIdx, 'field'), 4, '0');
-    const camcol= getCellValue(table, rowIdx, 'camcol');
-    const baseUrl= `${DAX_URL}/image/v0/calexp/ids`;
+    const deepCoaddId = Number(getCellValue(table, rowIdx, 'deepCoaddId'));
+    const searchIdBase = deepCoaddId - deepCoaddId%8;
+    const sr= new ServerRequest('LSSTImageSearch');
 
     return (plotId, title, filterName) => {
 
-        // id is run + filterId + camcol + field
-        // const objId= getCellValue(table, rowIdx, 'scienceCcdExposureId');
-        // const objId= `${run}${filterId}${camcol}${field}`;
-        // const url= `http://lsst-qserv-dax01.ncsa.illinois.edu:5000/image/v0/calexp/id?id=${objId}`;
-        const url= `${baseUrl}?run=${run}&camcol=${camcol}&field=${field}&filter=${filterName}`;
-        return makeUrlPlotRequest(url, title, plotId, rangeValues);
+        const searchId = searchIdBase + Number(bandMap[filterName]);
+        sr.setParam('deepCoaddId', `${searchId}`);
```

Comment:
there is a way to pass tract and patch and filter here.  I think we decided to do that instead.  This affects the search processor as well